testing/src/test/java/io/stargate/it/CQLTest.java

```diff
@@ -31,51 +34,53 @@
 import com.datastax.oss.driver.api.core.cql.Row;
 import com.datastax.oss.driver.api.core.cql.SimpleStatement;
 import com.datastax.oss.driver.api.core.data.TupleValue;
+import com.datastax.oss.driver.api.core.data.UdtValue;
 import com.datastax.oss.driver.api.core.metadata.schema.KeyspaceMetadata;
 import com.datastax.oss.driver.api.core.metadata.schema.TableMetadata;
 import com.datastax.oss.driver.api.core.servererrors.InvalidQueryException;
 import com.datastax.oss.driver.api.core.type.DataTypes;
 import com.datastax.oss.driver.api.core.type.TupleType;
+import com.datastax.oss.driver.api.core.type.UserDefinedType;
 import com.datastax.oss.driver.internal.core.loadbalancing.DcInferringLoadBalancingPolicy;
 import net.jcip.annotations.NotThreadSafe;
 
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.fail;
 
 @RunWith(Parameterized.class)
 @NotThreadSafe
 public class CQLTest extends BaseOsgiIntegrationTest
 {
-    private static final Logger logger = LoggerFactory.getLogger(PersistenceTest.class);
-
     @Rule
     public TestName name = new TestName();
 
     private String table;
     private String keyspace;
-    private CqlSession session;
+    private static CqlSession session;
 
     @Before
     public void setup()
     {
-        DriverConfigLoader loader =
-                DriverConfigLoader.programmaticBuilder()
-                        .withBoolean(DefaultDriverOption.METADATA_TOKEN_MAP_ENABLED, false)
-                        .withString(DefaultDriverOption.LOAD_BALANCING_POLICY_CLASS, DcInferringLoadBalancingPolicy.class.getName())
-                        .withDuration(DefaultDriverOption.REQUEST_TIMEOUT, Duration.ofSeconds(5))
-                        .withDuration(DefaultDriverOption.CONNECTION_INIT_QUERY_TIMEOUT, Duration.ofSeconds(5))
-                        .withDuration(DefaultDriverOption.CONTROL_CONNECTION_TIMEOUT, Duration.ofSeconds(5))
-                        .withDuration(DefaultDriverOption.REQUEST_TRACE_INTERVAL, Duration.ofSeconds(1))
-                        .build();
-
-        session = CqlSession.builder()
-                .withConfigLoader(loader)
-                .addContactPoint(new InetSocketAddress(stargateHost, 9043)).build();
-
         String testName = name.getMethodName();
         testName = testName.substring(0, testName.indexOf("["));
-        keyspace = "ks_" + testName;
+        keyspace = "ks_" + new Date().getTime() + "_" + testName;
         table = testName;
+    }
 
+    @BeforeClass
+    public static void beforeAll()
+    {
+        session = CqlSession.builder()
+            .withConfigLoader(getDriverConfigLoaderBuilder()
+                .withDuration(DefaultDriverOption.REQUEST_TRACE_INTERVAL, Duration.ofSeconds(1))
+                .build())
+            .addContactPoint(new InetSocketAddress(stargateHost, 9043)).build();
+    }
+
+    @AfterClass
+    public static void afterAll()
+    {
+        session.close();
```

Comment:
This can be addressed when we improve the test harness and how we create integration tests (https://datastax.jira.com/browse/C2-286). 

If its ok by you, I'll leave this suggestion out.